DungeonSource/Ogre.java

```diff
@@ -15,16 +15,16 @@
     public Ogre()
 	{
 		super("Oscar the Ogre", 200, 2, .6, .1, 30, 50, 30, 50);
-
+		this.healBehavior = new MonsterBasicHealBehavior();
+		this.attackBehavior = new BaseAttackBehavior();
```

Comment:
Later on possibly in the final project assignment we will have to add functionality to support multiple monster attack behaviors. For now this should be okay.